### PR TITLE
IGNORE_OVERLAP in telemetry

### LINKED ISSUE #1202: Packet level IGNORE_OVERLAP
We have the OVERLAP keyword to note that individual packet items can overlap. However if you have existing packet definitions with hundreds of items it can be painful to note each one. Create a packet level keyword called IGNORE_OVERLAP that simply ignores all overlap items across the entire packet.

### PR DESCRIPTION
**Pull request by jmthomas**
closes #1202 

Branched from #1201 because that introduced test_packet_config.py file

### CODE REVIEW

File: openc3/lib/openc3/packets/packet.rb [hunk 1]
```diff
@@ -151,12 +155,10 @@ def target_name=(target_name)
           if !(String === target_name)
             raise(ArgumentError, "target_name must be a String but is a #{target_name.class}")
           end
-
           @target_name = target_name.upcase.freeze
         else
           @target_name = nil
         end
-        @target_name
```
Comment (ryanmelt) [inline openc3/lib/openc3/packets/packet.rb:159]: I understand this is redundant, but I like explicit returns.   Should be "return @target_name"

**Review by ryanmelt — CHANGES_REQUESTED** [review]
Please add explicit returns to places where last line was removed.

File: openc3/lib/openc3/packets/packet.rb (hunk 1 above)
Comment (jmthomas) [reply openc3/lib/openc3/packets/packet.rb:159]: A setter should not have a return. That's why I removed it ... not because it was implicit

Comment (jmthomas) [reply openc3/lib/openc3/packets/packet.rb:159]: https://ruby-doc.org/3.2.2/syntax/methods_rdoc.html#label-Return+Values

**Pull request merged**